requirements.txt

```diff
@@ -6,7 +6,7 @@ appengine-python-standard>=1.0.0
 html5lib==1.1
 funcsigs==1.0.2
 google-api-python-client==2.47.0
-google-python-cloud-debugger==2.18
+# Only suppoted for py3.9: google-python-cloud-debugger==2.18
```

Comment:
Could you add an issue to revisit this in the future when 3.10 is supported. https://github.com/GoogleCloudPlatform/cloud-debug-python/issues/33